src/main.js

```diff
@@ -86,7 +105,6 @@ module.exports = {
           atom.notifications.addError('Linter-ESLint: Please save before fixing')
           return
         }
-
```

Comment:
Any reason this was cut? vertical whitespace itself isn't bad, just too much of it :wink: